tensorflow/lite/micro/kernels/BUILD

```diff
@@ -183,6 +187,24 @@ cc_library(
     }),
 )
 
+cc_library(
+    name = "xtensa",
+    hdrs = select({
+        "//conditions:default": [
+        ],
+        ":xtensa_hifimini": [
+            "xtensa/xtensa.h",
+        ],
+    }),
+    copts = micro_copts(),
+    deps = select({
+        "//conditions:default": [],
+        ":xtensa_hifimini": [
+            #"//third_party/xtensa/cstub64s:hifi_mini",
```

Comment:
What is the purpose of this commented line?

Reply:
This is automatically uncommented by copybara. Allows us the use of the proprietary cstubs internally while not breaking the build externally.